dwave/system/composites/embedding.py

```diff
@@ -494,6 +495,171 @@ def sample(self, bqm, chain_strength=1.0, chain_break_fraction=True, **parameter
         return super(LazyFixedEmbeddingComposite, self).sample(bqm, chain_strength=chain_strength,
                                                                chain_break_fraction=chain_break_fraction, **parameters)
 
+class ParamEmbeddingComposite(dimod.ComposedSampler):
```

Comment:
Thanks for looking at this!
The code duplication is on purpose, because my goal wasn't to replace the EC. But you're right, it would make more sense, going forward, to make something like ParamEC the class from which the other *EC build from.